web-frontend/src/pages/TransferCoins/ReceiveCoins/CopyContent.tsx

```diff
@@ -1,10 +1,10 @@
 import { FiCopy } from 'react-icons/fi';
 
-function CopyAddress({ ...props }) {
-  const { address, formattedAddress } = props;
+function CopyContent({ ...props }) {
+  const { content, formattedContent } = props;
```

Comment:
can you create the interface for CopyAddress props? with formattedContent type ?string